centaur/src/main/resources/standardTestCases/custom_entrypoint/custom_entrypoint.wdl

```diff
@@ -0,0 +1,25 @@
+version 1.0
+
+workflow custom_entrypoint_wf {
+    call custom_entrypoint_task
+    output {
+        String hello = custom_entrypoint_task.hello
+    }
+}
+
+task custom_entrypoint_task {
+    meta {
+        # Don't allow call-caching
+        volatile: true
+    }
+    command <<<
+        echo hello world
+    >>>
+    runtime {
+        # This image has a pre-defined entrypoint so we _have_ to override it:
+        docker: "denvazh/gatling:3.2.1"
```

Comment:
Is this image and tag stable enough to use as the foundation for our CI?

Reply:
I assume so - they'd have to go and retrospectively change history on github to break this... Happy to switch if there's a more appropriate candidate?

Reply:
This is *the* gatling perf-test tool image btw, not _just_ some random person's docker image

Reply:
Well... maybe not an official image but it does at least seem reasonably widely-used: 
![image](https://user-images.githubusercontent.com/13006282/90180417-ed96fd00-dd7c-11ea-91ec-da395f6b5b8f.png)

Reply:
we've historically put images [in a repo we control](https://github.com/broadinstitute/cromwell/blob/develop/centaur/src/main/resources/standardTestCases/non_root_default_user/non_root_default_user.wdl#L8) though I'm not sure it's worth the bother... 🙂